src/test/java/com.endava.internship.collections/StudentMapTest.java

```diff
@@ -0,0 +1,231 @@
+package com.endava.internship.collections;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.time.LocalDate;
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class StudentMapTest {
+
+    private final static List<Student> students = Stream.of(
```

Comment:
![image](https://user-images.githubusercontent.com/67113353/191904165-1269ab09-eced-4486-81ac-fc8ec82b37d8.png)
![image](https://user-images.githubusercontent.com/67113353/191904229-91e8ce7b-0673-427d-a063-245450e2bc2a.png)
![image](https://user-images.githubusercontent.com/67113353/191904254-5f72180c-818f-44ca-ac0f-3756d5fefdd1.png)
the way to fix it was this

Reply:
I got you. thanks